Maths/PrimeCheck.py

```diff
@@ -0,0 +1,14 @@
+def primeCheck(number):
+    prime = True
+    for i in range(2, int(number**(0.5)+1), 2):
+        if number % i == 0:
+            prime = False
+            break
+    return prime
+
+def main():
+    print(primeCheck(37))
+    print(primeCheck(100))
+
```

Comment:
Run the following:
```python
primeCheck(77)
```
and you will get **True** in return (*which as you know  is wrong because **77** is definitely not a prime number.*).

---
However, you can say that **77** is a **semiprime** as it is the product of two distinct prime numbers.

Reply:
Fixed